### ISSUE TITLE
Use assertEqual instead of deprecated assertEquals

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8625]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8625|
|Type|enhancement|
|Created|2016-07-18 20:22:13Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8625 8625
type__enhancement enhancement
reporter__scop scop
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__names names
keywords__None None
time__1468873333072835 1468873333072835
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_1"></a>|@scop commented|
|-|-|

https://github.com/twisted/twisted/pull/352

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 1069e367aa66123814f7ec0f993a11e7b6c0b5a3
```
#!CommitTicketReference repository="" revision="1069e367aa66123814f7ec0f993a11e7b6c0b5a3"
Merge pull request #352 from scop/8625-scop-assertequal

Author: scop
Reviewer: alex, rodrigc
Fixes: #8625

Use assertEqual instead of deprecated assertEquals
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
